src/osw_hal.cpp

```diff
@@ -86,10 +88,15 @@ void OswHal::stop(bool toLightSleep) {
     this->displayOff(); // This disables the display
     OswServiceManager::getInstance().stop();
 
-    if(!toLightSleep) {
+    if (!toLightSleep) {
+        for (int x = DISP_W-1; x>=0; --x)
+            for (int y = DISP_H-1; y>=0; --y)
+                this->gfx()->drawPixel(x, y, rgb565(0, 0, 0));
+
+        this->flushCanvas();
```

Comment:
1. why not use accelerated functions like fastVLine?
2. don't we have a toSleep() somewhere in the display, isn't that a more logical place to place this?
3. what is the performance if "low-memory"-mode (no buffering) is active? drawing pixel by pixel seems awfully slow